node/Cargo.toml

```diff
@@ -38,6 +39,7 @@ codec = { package = "parity-scale-codec", version = "3.0.0" }
 serde = { version = "1.0.137", features = ["derive"] }
 serde_json = "1.0.68"
 hex-literal = "0.3.1"
+tracing-core = "=0.1.26"
```

Comment:
Do you think we need a comment here to remember to clean it up in a future release?